### ISSUE TITLE
Thank You for your Effort

### ISSUE THREAD

**Issue by kreezxil** [sirdude/gurbalib#7]
Thanks for all the work you put into making GurbaLib work with DGD. While I don't have children, I have been wanting to get back to my own Mudding roots. You see I am actually Trimaster of Highlands I, II, and III original hosted at SHSU.EDU, no longer in existence.

The last mud was involved in was Revolution MUD back in 1999 where I was working on a daemon that allowed creators to dynamically build their areas. All they had to do was clone an object into a room and it would persist permanently thru reboots and crashes. Appropriately enabled objects were even movable by players and also retained their positions and other changed attributes. I assume it is something similar to the Tile Entities in Minecraft.

In simpler terms I had been working on a system where the creator did not have to know how to code. As long as they could pass config options to an object or room and clone the object and load a room, that was all that was needed.

This time around which is now almost 20 years later, because now I'm 51, I intend to take it further. I'll implement the daemon again and this time extend it to mobs and also give mobs an ai capability so that unattended mobs can grow their own communities and trade routes. This would not have been allowed back in the day because of the processor power required but with cheap massively powerful dedicated servers as provided by https://nocix.net and https://kimufi.com that fear is a thing of the past. We should just go for it. I know I am.

And yes feel free to browse my repositories as I have ton of mods for Minecraft 1.8.9 thru 1.12.2 and soon 1.13.

Also a ton of Modpacks, my latest claim to fame is World of Dragons. 
http://minecraft.curseforge.com/members/kreezxil/projects

Once again thank you for all your effort.

minecruft.kreezcraft.com 4000

**Comment by sirdude** [comment]
Hey thanks for the message.  I've been at training so was out all week, I'll take a look at your patch and get it added soon.  Great to see some interest.
